src/class-parsely-recommended-widget.php

```diff
@@ -115,6 +115,20 @@ public function widget( $args, $instance ) {
 
 		$reco_widget_script_asset = require PARSELY_PLUGIN_DIR . 'build/admin-page.asset.php';
 
+		?>
+
+		<div class="parsely-recommended-widget"
+			data-parsely-widget-display-author="<?php echo esc_attr( isset( $instance['display_author'] ) ? wp_json_encode( boolval( $instance['display_author'] ) ) : false ); ?>"
+			data-parsely-widget-display-direction="<?php echo esc_attr( isset( $instance['display_direction'] ) ? $instance['display_direction'] : null ); ?>"
```

Comment:
Instead of adding `null` string to the attribute, could an empty string be used instead? Just seems cleaner.